automated install/basic-install.sh

```diff
@@ -859,6 +858,9 @@ use4andor6
 # Decide what upstream DNS Servers to use
 setDNS
 
+# Configure SELinux (if applicable)
```

Comment:
I think this may be causing an issue where the install script is trying to copy files from `/etc/,pihole` to build the SEModules, but since `installPihole` hasn't been run yet, that directory hasn't been created?